src/main/java/com/ays/common/util/exception/model/AysError.java

```diff
@@ -113,7 +113,12 @@ public enum Header {
         /**
          * AUTH_ERROR header.
          */
-        AUTH_ERROR("AUTH ERROR");
+        AUTH_ERROR("AUTH ERROR"),
+
+        /**
+         * FORBIDDEN_ERROR header.
+         */
+        FORBIDDEN_ERROR("FORBIDDEN ERROR");
```

Comment:
@ilkerCelimli, we can remove the header here and use the `AUTH_ERROR` header in the same way we use it in the application, even in the test side.